### ISSUE TITLE
Update Accessibility Conformance Documentation

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4302]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark;  #JavatoolingforIntelliJ/Eclipse; #C+E-H; #A11ySev1; #A11yAAT 

**Recommendation:** "Fix and close other bugs before starting accessibility conformance documentation. Any bug that will not be fixed and closed within SLA must be declared in the AAT. The SLA for this bug is also 60 days."

Link to the current AAT entry - https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076

**Comment by Flanker32** [comment]
@soumyatippapuram Thanks for your help, we have fixed all the accessibility issues and linked the external issues to AAT(except the WCAG one), could we close this issue now? Thanks!

**Comment by soumyatippapuram** [comment]
@Flanker32 I have reviewed the AAT and found the below changes:

MAS Name | MAS ID | Marked in AAT   as | Should be   changed to
-- | -- | -- | --
Error   Prevention (Legal, Financial, Data) | MAS 3.3.4 | Supported | Not Applicable
Character Key   Shortcuts | MAS 2.1.4 | Supported | Not Applicable
Pointer   Cancellation | MAS 2.5.2 | Not Applicable | Supported
Label in Name | MAS 2.5.3 | Supported | Supported with   Exceptions


This bug can be closed once the above changes are made in AAT

**Comment by Flanker32** [comment]
@soumyatippapuram Thanks for the info, i've updated the AAT document, please have a review.

**Comment by Flanker32** [comment]
@soumyatippapuram  Thanks for your help, we have linked all the accessibility issues to AAT and change the MAS status, could you please have a review and close this issue?

**Comment by soumyatippapuram** [comment]
@Flanker32 I will review it and update the status shortly

**Comment by soumyatippapuram** [comment]
@Flanker32 I have reviewed the AAT. Everything is marked correctly. You can proceed with next steps.
This bug will be closed once the AAT is published.

**Comment by soumyatippapuram** [comment]
The AAT is Finalized. Hence, closing this.
![Bug_4302_Java Tooling Benchmark_AAt_Finalized](https://user-images.githubusercontent.com/41108985/87298362-b1773f00-c527-11ea-9943-13ff3dcb8e5c.png)
